projects/packages/status/src/class-status.php

```diff
@@ -45,8 +45,7 @@ public function is_offline_mode() {
 		/**
 		 * Filters Jetpack's offline mode.
 		 *
-		 * @see https://jetpack.com/support/development-mode/
-		 * @todo Update documentation ^^.
+		 * @see https://jetpack.com/support/offline-mode/
```

Comment:
Good change, but not related to old PHP or WP versions.

Reply:
Updated PR title.